src/org/labkey/targetedms/TargetedMSModule.java

```diff
@@ -507,6 +513,24 @@ protected void init()
         addController("targetedms", TargetedMSController.class);
         addController("passport", PassportController.class);
 
+        Portal.registerNavTreeCustomizer("Targeted MS QC Plots", viewContext -> {
+            List<NavTree> result = new ArrayList<>();
+            if (viewContext.hasPermission(AdminPermission.class))
+            {
+                NavTree save = new NavTree("Save as Default View");
+                save.setScript("LABKEY.targetedms.PlotSettingsUtil.saveAsDefault()");
+                result.add(save);
+            }
+            if (!viewContext.getUser().isGuest())
```

Comment:
This means even the Readers would be able to revert to default, but RevertToDefaultQCPlotSettingsAction requires AdminPermission throwing permission error.

Reply:
Thanks. I forgot to update the annotations on RevertToDefaultQCPlotSettingsAction after I copy/pasted the code to make the action. Fixed.